docker-compose.yml

```diff
@@ -46,19 +46,21 @@ services:
       - "2181:2181"
 
   elasticsearch:
-    image: elasticsearch:latest
+    image: elasticsearch:2.3.1
```

Comment:
I would prefer to just use ELK 5.0 in local testing, since we are moving to that soon anyway